id/__main__.py

```diff
@@ -44,6 +45,12 @@ def _parser() -> argparse.ArgumentParser:
         default=0,
         help="run with additional debug logging; supply multiple times to increase verbosity",
     )
+    parser.add_argument(
+        "-d",
+        "--decode",
+        action="store_true",
+        help="decode the OIDC token into JSON",
+    )
```

Comment:
Extreme nitpick: the help here implies (to me) that doing `python -m id foo --decode | jq` would produce a single JSON streams, but the actual implementation prints two separate JSON streams (for the header and payload).

I think this barely matters in practice, though, since someone will probably just manually comb through these.

Reply:
That makes sense to me, and also doing that would necessitate removing the signature from the output, which I've decided to do: cc720bd